pkg/types/types.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+// TODO: Add more configs.
+type Parameter struct {
```

Comment:
is this user facing?  if so, let's make it a versioned package?

Reply:
It is not user-facing. the magic commands are user facing and I think we should make that a versioned package in the future (After we have a real interpreter.)